index.js

```diff
@@ -22,11 +22,13 @@ const pThrottle = ({limit, interval}) => {
 	let activeCount = 0;
 
 	return function_ => {
+		// Uses function and context so that this function is bound to calling context's `this`
 		const throttled = function (...args) {
+			const context = this;
 			let timeout;
 			return new Promise((resolve, reject) => {
 				const execute = () => {
-					resolve(function_.apply(this, args));
+					resolve(function_.apply(context, args));
```

Comment:
I'm not interested in changing my code to make it better for transpilation.

Reply:
I dropped the commits that strengthened the code for transpilation.